main.py

```diff
@@ -122,12 +216,14 @@ def _is_video_tweet(embed: discord.Embed) -> bool:
     if not is_twitter_embed:
         return False
     if embed.image.url is None:
-        return True
+        return False
 
     # native video embed now uses a thumbnail instead of broken video
     # URL begins with https://pbs.twimg.com/ext_tw_video_thumb/
+    # or https://pbs.twimg.com/tweet_video_thumb/
     contains_video_thumbnail = "ext_tw_video_thumb" in embed.image.url
-    return contains_video_thumbnail
+    contains_gif_thumbnail = "tweet_video_thumb" in embed.image.url
+    return contains_video_thumbnail or contains_gif_thumbnail
```

Comment:
Is there a reason this is not one single return statement with both `in` checks inlined?
Or, for that matter, a simple `"_video_thumb/" in embed.image.url`?
Any applicable delimiting slashes are likely good to include as well, whether we have a "full" check or a slightly simpler one.

Reply:
True, swapped to this